test/browser/fragments.test.js

```diff
@@ -1172,29 +1237,22 @@ describe('Fragment', () => {
 		render(<Foo condition={false} />,  scratch);
 		expect(scratch.innerHTML).to.equal(htmlForFalse, 'rendering from true to false');
 		expectDomLogToBe([
-			// see issue #193 - re-appends all children after Fragment
-			'<ol>002345.appendChild(<li>0)',
-			'<ol>013450.appendChild(<li>1)',
-			'<ol>024501.appendChild(<li>2)',
-			'<ol>345012.appendChild(<li>3)'
+			'<ol>012345.insertBefore(<li>4, <li>0)',
+			'<ol>401235.insertBefore(<li>5, <li>0)',
+			// TODO: see issue #193 - Hmmm why does this extra append happen?
```

Comment:
TODO: Migrate issue 193 from old repo and integrate with issue mentioned in description